### ISSUE TITLE
[Bug 🐞]: Files open in Webview

### ISSUE THREAD

**Issue by MatsG23** [davidsmorais/kuro#94]
**Describe the bug**

**Reproduction Steps**
1. Add a file to some task
2. Click on the file in Kuro
3. A web view is opened as a new window
4. The Outlook page never stops loading
5. The file appears in the Downloads folder

**Expected Behavior**
I expect the file to open directly on the computer like it did some time ago.

**Screenshots**
![screenshot_kuro](https://user-images.githubusercontent.com/60609373/226115873-6bca999a-bcca-45b5-8053-8e108c3cb871.png)

**Technical Info (please complete the following information)**
 - OS: openSUSE Tumbleweed 20230316
 - DE: KDE Plasma 5.27.2 (Wayland) 
 - Kuro Version: 9.0.0

---
cc:
@davidsmorais

**Comment by davidsmorais** [comment]
Hey @MatsG23 , thanks for reporting this.
I'll try and have a fix for this issue as soon as this week

**Comment by davidsmorais** [comment]
Quick update on this:

It seems MS ToDo changed its behavior and when you click on an attachment it now opens an Outlook deeplink to your image. 
However, this feature seems to be currently bugged on the MS ToDo web client as the Outlook page had an error when I tried to access the deeplink for the first time after logging in the browser/Kuro.

In this case, Kuro will open the attachment in another Kuro window pointed to the Outlook deeplink :point_down:
 
![image](https://user-images.githubusercontent.com/22729436/226768606-1c612d3f-aba0-4056-9aee-13dcc43ee34a.png)

Basically, this change was done by Microsoft and there's nothing I can do here to revert the behavior back to how it was. Hopefully they will fix the deeplink not working at the first try, but for the moment, Kuro is mimicking what's done on the official web client
